ChatDemo-UI3.0/ChatDemo-UI3.0/Class/Chat/ChatList/ConversationListController.m

```diff
@@ -61,11 +61,12 @@ - (void)viewDidLoad {
     
     [self tableViewDidTriggerHeaderRefresh];
     
-    [self.view addSubview:self.searchBar];
-    self.tableView.frame = CGRectMake(0, self.searchBar.frame.size.height, self.view.frame.size.width, self.view.frame.size.height - self.searchBar.frame.size.height);
+//    [self.view addSubview:self.searchBar];
+//    self.tableView.frame = CGRectMake(0, self.searchBar.frame.size.height, self.view.frame.size.width, self.view.frame.size.height - self.searchBar.frame.size.height);
+    
     [self networkStateView];
     
-    [self searchController];
```

Comment:
search bar on the chats screen right now only does the search for existing contact on the page, cause more confusion than being helpful.